src/RouteCollector.php

```diff
@@ -164,9 +164,13 @@ private function checkForDuplicateRoute(string $path, array $methods = null) : v
         });
 
         if (! empty($matches)) {
-            throw new Exception\DuplicateRouteException(
-                'Duplicate route detected; same name or path, and one or more HTTP methods intersect'
-            );
+            $match = reset( $matches );
+            throw new Exception\DuplicateRouteException(sprintf(
+                'Duplicate route detected; "%s : %s : %s"',
+                implode( ',', $match->getAllowedMethods() ?? [] ),
```

Comment:
170 | ERROR | [x] Space after opening parenthesis of function call prohibited
 170 | ERROR | [x] Expected 0 spaces before closing bracket; 1 found

Same here, no spaces issues.